File: test/items/example/__check/ts/auth/auth-model.ts




/**
* User model base
**/
export interface UserBase
{
   /**
   * Required. 
   **/
  UserName: string;
   /**
   * Required. ads
   **/
  Psw: boolean;
}
/**
* User model
**/
export interface User extends UserBase 
{
   /**
   * Optional. 
   **/
  EMail: string;
}

File: test/items/example/__check/cs/auth/auth-model.cs
using System;

namespace Example.Auth
{


  /// <summary>
  /// User model base
  /// </summary>
  public partial class UserBase
  {
    /// <summary>
    /// Required. 
    /// </summary>
    public virtual string UserName {get; set; }
    /// <summary>
    /// Required. ads
    /// </summary>
    public virtual bool Psw {get; set; }
  }
  /// <summary>
  /// User model
  /// </summary>
  public partial class User : UserBase 
  {
    /// <summary>
    /// Optional. 
    /// </summary>
    public virtual string EMail {get; set; }
  }


}
